subprojects/core/src/integTest/groovy/org/gradle/api/ApplyPluginIntegSpec.groovy

```diff
@@ -39,7 +39,6 @@ class ApplyPluginIntegSpec extends AbstractIntegrationSpec {
 
     @Issue("GRADLE-2358")
     @FailsWith(UnexpectedBuildFailure)
-    // Test is currently failing
```

Comment:
Can we move the comment back to where it was before? Behind: `@FailsWith(UnexpectedBuildFailure)`